src/components/common/Loading.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import Spinner from './icons/Spinner';
+import { Header, Container } from './Layout';
+import { Title, Paragraph } from './Typography';
+
+const Spacer = styled.div`
+  padding-top: 6rem;
+`;
+
+const title1 = 'Processing....';
+const para1a = 'We are currently analysing the data to extract the relevant medical entities.';
+const para1b = 'This can take a few minutes so please do not refresh this page.';
+
+const title2 = 'Finding Clinical Trials....';
+const para2a = 'We are currently filtering Clinical Trials across the UK to find matches for each of the entries provided.';
+const para2b = 'This can take a few minutes so please do not refresh this page.';
```

Comment:
title1, title2, para1a, para2b should be two props passed by (title) 
also, `para2b` and `para1b` is the same sentence so could be defined as one variable 
or could leave them and check for them like the comment below